no-std-libc-demo/x86_64-posix-elf.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
For the uninitiated, how do you arrive at a file like this? The README mentions this is "an OS-agnostic version of `x86_64-unknown-linux-gnu`", does that mean you source the `x86_64-unknown-linux-gnu` target configuration from upstream `rustc`? If so maybe link that in the README. It would also be helpful to know what needed to be customized to achieve agnosticism compared to standard.